Source/cryptalgo/CMakeLists.txt

```diff
@@ -17,14 +17,21 @@
 
 set(TARGET  ${NAMESPACE}Cryptalgo)
 
-add_library(${TARGET} SHARED
+set(SOURCES
```

Comment:
please use target_sources instead of introducing a variable...
https://cmake.org/cmake/help/v3.15/command/target_sources.html

Reply:
updated